examples/ManyModel.py

```diff
@@ -243,6 +243,11 @@ def main():
     parser.set_defaults(channels_last=False)
 
     args = parser.parse_args()
+    if args.lms_stats or args.lms_stats_average:
```

Comment:
Per the scrum discussion, I vote for creating the directory if it doesn't already exist.